src/editions.md

```diff
@@ -0,0 +1,39 @@
+# Editions
+
+Before we get into the details of the language itself, we need to cover the
+concept of an "edition."
+
+An "edition" of Rust consists of a particular year. There are two editions of
+Rust, currently:
+
+* Rust 2015
+* Rust 2018
+
+Conforming compilers are expected to:
```

Comment:
The reference isn't a specification (even if it's the closest to a specification we have), so we can't say what a conforming compiler is from that perspective and we don't have a specification. So we don't really have a definition of what a conforming compiler can be outside the reference and we don't want to define that here either. We can talk about "The compiler" referring to `rustc`. We're pretty much pretending no other compiler exists at this time.